source/codegen/templates/service_helpers.mako

```diff
@@ -269,21 +290,16 @@ ${initialize_standard_input_param(function_name, parameter)}\
 <%
   parameter_name = common_helpers.camel_to_snake(parameter['cppName'])
   underlying_param_type = common_helpers.get_underlying_type_name(parameter["type"])
+  c_element_type_that_needs_coercion = service_helpers.get_c_element_type_for_array_that_needs_coercion(parameter)
```

Comment:
I don't think `c_element_type_that_needs_coercion` is used in this `def`. I would've expected it to be used after `is_output_array_that_needs_coercion` was checked to mirror what's done for initializing the input params but I might be missing something.